### ISSUE TITLE
check out issues from shopify to viva wallet

### ISSUE THREAD

**Issue by fkarelas** [VivaPayments/API#1330]
after connecting shopify with viva wallet using the token at check out i get a message error input is invalid. This is not a new account but an existing online store that was accessible only in the US and now has migrated to the EU. 

Thank you in advance
![viva](https://user-images.githubusercontent.com/60217963/73004421-eecc0100-3e0f-11ea-8c74-66cd08a0532c.JPG)

**Comment by matthewrgourd** [comment]
Hi @fkarelas, sorry to hear you've been experiencing difficulties setting up Shopify. If you untick the 'test mode' option your store should be found at the point of redirect. Please let me know how you get on. Thanks!

**Comment by fkarelas** [comment]
Dear @matthewrgourd thanks for your answer. I have already unchecked test mode before writing the issue on this board. Any other ideas?

**Comment by fkarelas** [comment]
@matthewrgourd after finishing checkout it does not take us to viva wallet page to input credit card details

**Comment by matthewrgourd** [comment]
Hi @fkarelas this means Viva Wallet for Shopify hasn't been set up correctly. Please follow our [guide](https://developer.vivawallet.com/e-commerce-plugins/shopify/) verbatim to ensure each step is covered. Thanks!

**Comment by fkarelas** [comment]
dear @matthewrgourd the steps described are very different than what i am aloud to do, please help us the we are loosing funds everyday, this is the 6th day trying to figure this out

**Comment by matthewrgourd** [comment]
@fkarelas please elaborate on what you mean by "This is not a new account but an existing online store that was accessible only in the US and now has migrated to the EU" as the problem may be something related to this. Are you still using the US version of Shopify?

**Comment by fkarelas** [comment]
Dear @matthewrgourd the issue has been resolved, spoke to viva wallet dustomer service and clarified the issue as there was a wrong merchant id place, we used the viva wallet id instead of the merchant id which is more difficult to find in the account info. Now works like a charm, thank you for your effort. You can close the ticket if you wish or i can do it.

**Comment by fkarelas** [comment]
Also for your information and any future need. This was an online store set up in the US with US merchant accounts and billing and decided to move to Europe. We deactivated the US merchant accounts and payment providers and changed currency in coordination with shopify customer service. After doing that i tried to activate viva wallet payment but could not do it because of the wrong merchant id input from us. We corolated the problem with the already existing e-shop but it turns out that was not the issue. Thank you again.
